Extend Layers Control tutorial to add layer dynamically

QuentinPhilipp opened the pull request:
Hello! 

This PR resolves #7787 by adding an example how to add a base layer and an overlay dynamically in a Layers Control. 

I've also updated the code for the live example.

Falke-Design commented on docs/SlavaUkraini/examples/layers-control/index.md:
```diff
@@ -54,19 +54,32 @@ var overlayMaps = {
+Finally, you can add or remove base layers or overlays dynamically:
+
+<pre><code>var crownHill = L.marker([39.75, -105.09]).bindPopup('This is Crown Hill Park.'),
+    rubyHill = L.marker([39.68, -105.00]).bindPopup('This is Ruby Hill Park.')
```
```suggestion
    rubyHill = L.marker([39.68, -105.00]).bindPopup('This is Ruby Hill Park.');
```

Falke-Design commented on docs/SlavaUkraini/examples/layers-control/example.md:
```diff
@@ -32,4 +32,11 @@ title: Layers Control Tutorial
 
 	var layerControl = L.control.layers(baseLayers, overlays).addTo(map);
+	var crownHill = L.marker([39.75, -105.09]).bindPopup('This is Crown Hill Park.'),
+	    rubyHill = L.marker([39.68, -105.00]).bindPopup('This is Ruby Hill Park.')
```
```suggestion
	    rubyHill = L.marker([39.68, -105.00]).bindPopup('This is Ruby Hill Park.');

```

Falke-Design reviewed (changes requested):
Please add missing semicolon, else it looks good. 
![grafik](https://user-images.githubusercontent.com/19800037/158335024-ee7a2fb3-b9b6-4a3e-b27d-869d7ac83c73.png)

QuentinPhilipp commented:
I approved proposed fix

QuentinPhilipp commented:
@Falke-Design I've noticed an issue in the code, the provided example I gave couldn't work since the name of the control layer wasn't correct. I fixed it and updated the thumbnail to match the new example as well.

Falke-Design reviewed (changes requested):
Please "zoom into" the picture, like before and the width should be 750px

![grafik](https://user-images.githubusercontent.com/19800037/158885147-9cab0364-9bd1-44e1-8aec-1662607e488c.png)

QuentinPhilipp commented:
I changed the size and zoomed into the picture. I don't know why the grayscale tile isn't rendering exactly the same as the previous example, I used the same map ID, maybe the map theme changed a little from the last time this example was made.

Falke-Design reviewed (approved):
Very nice. Thank you for your contribution!

QuentinPhilipp commented:
The thumbnail has been compressed from 157KB to 84KB

Falke-Design commented:
Thank you @QuentinPhilipp
